app/sagas/login.js

```diff
@@ -85,8 +85,18 @@ const handleLogout = function* handleLogout() {
 	}
 };
 
-const handleSetUser = function handleSetUser({ user }) {
+const handleSetUser = function* handleSetUser({ user }) {
```

Comment:
K.. so the one of problem that @kareemhamdy100  was facing is that when first time you open app after changing the language it is showing old language but then after you open it again it will show correct language soo problem lies https://github.com/RocketChat/Rocket.Chat.ReactNative/blob/944e4198bbaf95e78b2be6b6a1270029f0fa226a/app/sagas/selectServer.js#L40 see it is setting user settings from local storage the whole login and changing of internal setting happens after that (this internal storage is changed only on login ) so I added condition to change this on change of user setting

Reply:
@diegolmello without it Language change will still have bugs ..

Reply:
I am opening a issue for this